src/lwc/bdiObjectMappingModal/bdiObjectMappingModal.js

```diff
@@ -526,9 +529,10 @@ export default class bdiObjectMappingModal extends LightningElement {
     refreshImportRecordFieldOptions() {
         this.diImportRecordFieldOptions = [];
         this.diImportRecordStatusFieldOptions = [];
-
+        console.log('in refreshImportRecordFieldOptions namespace Wrapper is: ' + this.namespaceWrapper);
         let fieldsToExclude = this.getAlreadyMappedDIFields(); 
-        if (fieldsToExclude && this.dataImportFieldData) {
+        
+        if (fieldsToExclude && this.dataImportFieldData && this.namespaceWrapper) {
```

Comment:
Doesn't the call to this.getAlreadyMappedDIFields() on line 533 also reference "this.namespaceWrapper" and it's also outside of this if statement? There were a couple variations of the error message that showed up, and the one that reads "cannot read property 'currentNamespace' of undefined" I think might not be protected by the new conditional.